src/view/ChewingEditor.cpp

```diff
@@ -128,6 +130,57 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString caption = QString("<h3>%1</h3><p>%2</p>")
```

Comment:
:+1:  for the work!

`QString("<h3>%1</h3><p>%2 %3 (%4)</p>")`
the reason behind this change is that it will make the translation job less likely to screw up.